src/metabase/util/snake_hating_map.clj

```diff
@@ -71,4 +71,4 @@
        (vary-meta assoc :metabase.driver/metadata-type :metabase.driver/metadata-type.mlv2)
        ->SnakeHatingMap))
   ([k v & more]
-   (snake-hating-map (into {} (cons [k v] (partition-all 2 more))))))
+   (snake-hating-map (into {k v} (partition-all 2) more))))
```

Comment:
Bug. I added a test for this.

Reply:
Wow, this is a tricky fix. (The new line is what I find natural, but it's difficult to see how this fixes what bug.)